test/deploy-test-operator.sh

```diff
@@ -11,11 +11,11 @@ CATALOG_IMG=$2
 
 # Create storage classes.
 case $(kubectl get nodes -o yaml) in
-  *"attachable-volumes-gce-pd"*)
```

Comment:
This condition does't work, so skipped completely in current flow.
Updated the condition